scripts/systemtap/kem/kemd.c

```diff
@@ -150,48 +153,56 @@ int kemd_rbs_init(struct kem_rb *rbs)
 static int kemd_init_module(void)
 {
 	dev_t devno;
-	int   err = 0;
+	int err = 0;
 
 	kemd_nr_cpus = num_online_cpus();
 
 	printk(KERN_INFO "Number of CPUs: %d.\n", kemd_nr_cpus);
 
 	kemd_rbs = kzalloc(kemd_nr_cpus * sizeof(*kemd_rbs), GFP_KERNEL);
-	if (kemd_rbs == NULL) {
+	if (kemd_rbs == NULL)
+	{
 		printk(KERN_WARNING "Can't allocate kemd_rbs.\n");
 		return -ENOMEM;
 	}
 	err = kemd_rbs_init(kemd_rbs);
-	if(err != 0) {
+	if (err != 0)
+	{
 		printk(KERN_WARNING "Can't allocate kemd_rbs.\n");
 		goto out_rbs;
 	}
 	kemd_devices = kzalloc(kemd_nr_cpus * sizeof(*kemd_devices),
-			       GFP_KERNEL);
-	if (kemd_devices == NULL) {
+						   GFP_KERNEL);
+	if (kemd_devices == NULL)
+	{
 		printk(KERN_WARNING "Can't allocate mem for kemd_devices.\n");
 		goto out_rbs_inited;
 	}
 
-	if (kemd_major) {
+	if (kemd_major)
+	{
 		devno = MKDEV(kemd_major, kemd_minor);
 		err = register_chrdev_region(devno, kemd_nr_cpus,
-					     KEMD_DEV_NAME);
-	} else {
+									 KEMD_DEV_NAME);
```

Comment:
Apparently, the alignment of this line is wrong.

Reply:
Formatted  by putting them by it own line